File: 10_Introduction-Algorithms/praktikum/Problem2.py
def lampuTombol(n):
    lampu = False
    for i in range(1, (n + 1)):
        if n % i == 0:
            lampu = not lampu

    if lampu is True:
        return print("Lampu menyala")
    else:
        return print("Lampu mati")


n = input("Masukkan nilai n : ")
lampuTombol(int(n))

File: 10_Introduction-Algorithms/praktikum/Problem2.js
function lampuTombol(n) {
  let lampu = false;
  for (let i = 0; i < n + 1; i++) {
    if (n % i === 0) {
      lampu = !lampu;
    }
  }
  if (lampu) {
    return console.log("lampu menyala");
  } else {
    return console.log("lampu mati");
  }
}

lampuTombol(0);
